deploy/.env.example

```
@@ -11,6 +11,7 @@ LANGFLOW_LOG_LEVEL=debug
 LANGFLOW_SUPERUSER=superuser
 LANGFLOW_SUPERUSER_PASSWORD=superuser
 LANGFLOW_NEW_USER_IS_ACTIVE=False
+LANGFLOW_ENABLE_PUBLIC_SIGNUP=True
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LANGFLOW_ENABLE_PUBLIC_SIGNUP key should go before the LANGFLOW_LOG_LEVEL key

(UnorderedKey)